qemu/tests/virtio_fs_share_data.py

```diff
@@ -238,6 +241,14 @@ def get_viofs_exe(session):
                 if md5_guest != md5_host:
                     test.fail('The md5 value of host is not same to guest.')
 
+                error_context.context("Folder test under %s inside "
```

Comment:
```suggestion
                if cmd_new_folder and cmd_copy_file and cmd_del_folder:
                    error_context.context("Folder test under %s inside "
```
There will raise an error if some scenarios don't need to add folder testing.

Reply:
I consider folder test into the common test scenario like file dd testing.So I just add it to "if cmd_dd:" scenario.

Reply:
Yes, I got your point. But you put this part into block `if cmd_dd:`, meaning that it is related to dd testing.
But from my understanding, there is no relation between adding folder testing and dd testing, they are separate testing.
So I think we should put adding folder testing out of dd testing.
```
if cmd_dd:
...
if cmd_new_folder and cmd_copy_file and cmd_del_folder:
...
if cmd_xx:
...
```
How about @zhenyzha ?

Reply:
@YongxueHong  I agree with you, since we have expanded many cases in `virtio_fs_share_data.py`, I think it is necessary to distinguish the steps.

Reply:
It's ok for me.

git am 2825.patch 
Applying: virtio_fs: Add folder test in shared directory

```
 (1/2) Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.5.0.aarch64.io-github-autotest-qemu.virtio_fs_share_data.with_cache.auto.arm64-pci: PASS (174.15 s)
 (2/2) Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.5.0.aarch64.io-github-autotest-qemu.virtio_fs_share_data.with_cache.auto.with_reboot.arm64-pci: PASS (249.64 s)
```
LGTM.Acked-by: Zhenyu Zhang <zhenyzha@redhat.com>